_tuning-your-cluster/availability-and-recovery/remote-store/migrating-to-remote.md

```diff
@@ -24,9 +21,9 @@ Rolling upgrades, sometimes referred to as *node replacement upgrades*, can be p
 
 Review [Upgrading OpenSearch]({{site.url}}{{site.baseurl}}/upgrade-opensearch/index/) for recommendations about backing up your configuration files and creating a snapshot of the cluster state and indexes before you make any changes to your OpenSearch cluster.
 
-Before migrating to remote-backed storage, upgrade to OpenSearch 2.14.
+Before migrating to remote-backed storage, upgrade to OpenSearch 2.15.
 
-As of OpenSearch 2.14, OpenSearch nodes cannot be migrated back to document replication. If you need to revert the migration, then you will need to perform a fresh installation of OpenSearch and restore the cluster from a snapshot. Take a snapshot and store it in a remote repository before beginning the upgrade procedure.
+As of OpenSearch 2.15, OpenSearch nodes cannot be migrated back to document replication. If you need to revert the migration, then you will need to perform a fresh installation of OpenSearch and restore the cluster from a snapshot. Take a snapshot and store it in a remote repository before beginning the upgrade procedure.
```

Comment:
```suggestion
As of OpenSearch 2.15, nodes cannot be migrated back to document replication. To revert, perform a fresh installation and restore from a snapshot. Take a snapshot and store it remotely before upgrading.
```

Reply:
`Storing it remotely` doesn't spell out need to do a snapshot .